OptimizelySDK/Utils/EventTagUtils.cs

```diff
@@ -68,63 +68,75 @@ public static object GetRevenueValue(Dictionary<string, object> eventTags, ILogg
 
         public static object GetNumericValue(Dictionary<string, object> eventTags, ILogger logger  = null)
         {
-            string debugMessage = string.Empty;
-            bool isCasted = false;
-
             float refVar = 0;
+            bool isCasted = false;
+            string logMessage = string.Empty;
+            LogLevel logLevel = LogLevel.INFO;
 
             if (eventTags == null)
-                debugMessage = "Event tags is undefined.";
-
+            {
+                logMessage = "Event tags is undefined.";
+                logLevel = LogLevel.DEBUG;
+            }
             else if (!eventTags.ContainsKey(VALUE_EVENT_METRIC_NAME))
-                debugMessage = "The numeric metric key is not in event tags.";
-
+            {
+                logMessage = "The numeric metric key is not in event tags.";
+                logLevel = LogLevel.DEBUG;
+            }
             else if (eventTags[VALUE_EVENT_METRIC_NAME] == null)
-                debugMessage = "The numeric metric key value is not defined in event tags.";
-
+            {
+                logMessage = "The numeric metric key value is not defined in event tags.";
+                logLevel = LogLevel.ERROR;
```

Comment:
This should also be DEBUG

Reply:
Done. Please let me know if the same change needs to be done for GetRevenueValue.

Reply:
Yes, please also make the change for `GetRevenueValue`